doc/users/cmake_dealii.html

```diff
@@ -894,6 +895,44 @@ <h3> Selecting a compiler </h3>
     </p>
 
 
+    <a name="compileccache"></a>
+    <h3> Compiling using CCache </h3>
+    <p>
+      It is possible for the library to be built using the
+      <a href="https://ccache.dev/">CCache</a> ("compiler cache") tool.
+      This may be useful for those who frequently rebuild the library, as it
+      caches the output of the compilation (the C++ components, at least).
+      The next time that the library is built, if the compilation objects remain
+      the same then rebuilding them can be avoided as the results will be taken
+      from directly the cache. Anecdotally, CCache is not well suited to
+      everybody's workflow, so it may take some experimentation to determine if
+      it's effective and helpful, or offers no particular gain to you.
+    </p>
+
+    <p>
+      Building with CCache can be enabled directly via the command line. For
+      example, if building using GCC then
+<pre class="cmake">
+  CC="ccache gcc" CXX="ccache g++"
+
+  cmake -DCMAKE_C_COMPILER="ccache gcc" -DCMAKE_CXX_COMPILER="ccache g++" &lt;...&gt;
+</pre>
+      would indicate that ccache should invoke the GCC compilers to build deal.II.
+      CCache itself is invoked when calling <code>make</code> or
+      <code>ninja</code>.
+    </p>
+
+    <p>
+      When using a version of CMake that is greater or equal to 3.4, then using
+      CCache is a little easier. The above may be compressed into a single
+      configure-time argument
+<pre class="cmake">
+  cmake -DCXX_COMPILER_LAUNCHER="ccache" &lt;...&gt;
+</pre>
+      which ensures that CCache is invoked whenever C++ code is compiled.
+    </p>
```

Comment:
I think we should switch the order of these two paragraphs so that users see the official version first. I would also clarify with `CC="ccache gcc"` that that approach is only necessary with older versions of CMake.

Reply:
Good suggestion, thanks. I'll swap these two paragraphs around.